### PR TITLE
fix: Fix top navigation view foreground

### LINKED ISSUE #1317: NavigationView's selected option disappears when PaneDisplayMode is setted as Top
## Current behavior

When selecting an option on the NavigationView control, when the PaneDisplayMode is setted as Top, the selected option disappears. Doesn't show up.
Is it possible that same color as background is being setted?

## Expected behavior

The selected option should be visible.

## How to reproduce it (as minimally and precisely as possible)

1 - Open the attached project and compile it. 
2 - Execute the Windows, WPF & WASM projects. Not necessary to debug.
3 - You'll see two NavigationViews, one has the PaneDisplayMode as Left and the other one as Top.
4 - Select an option and you'll se that doesn't show up on the PaneDisplayMode Top one.

[1317-NavigationViewSelectionDisappears.zip](https://github.com/unoplatform/Uno.Themes/files/13860818/1317-NavigationViewSelectionDisappears.zip)

## Environment

Nuget Package:
NuGet package(s): Uno.WinUI, Uno.Material.WinUI

Package Version(s):
NuGet package version(s): Uno.WinUI 5.0.48, Uno.Material.WinUI 4.0.6

Affected platform(s):
- [ ] iOS
- [ ] Android
- [X] WebAssembly
- [X] UWP
- [ ] MacOS

## Anything else we need to know?

### PR DESCRIPTION
**Pull request by Youssef1313**
﻿GitHub Issue: closes #1317

## PR Type

What kind of change does this PR introduce?

- Bugfix
- Feature
- Code style update (formatting)
- Refactoring (no functional changes, no api changes)
- Build or CI related changes
- Documentation content changes
- Other... Please describe:

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [ ] Tested the changes where applicable:
	- [ ] UWP
	- [ ] iOS
	- [ ] Android
	- [ ] WASM
	- [ ] MacOS
- [ ] Updated the documentation as needed:
	- [ ] [General Doc Update](https://github.com/unoplatform/Uno.Themes/tree/master/doc)
	- [ ] [material-controls-styles.md](https://github.com/unoplatform/Uno.Themes/blob/master/doc/material-controls-styles.md)
	- [ ] [cupertino-controls-styles.md](https://github.com/unoplatform/Uno.Themes/blob/master/doc/cupertino-controls-styles.md)
	- [ ] [lightweight-styling.md (LightWeight Styling Resource Keys)](https://github.com/unoplatform/Uno.Themes/blob/master/doc/lightweight-styling.md)
- [ ] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

**Comment by kazo0** [comment]
might conflict/be addressed by https://github.com/unoplatform/Uno.Themes/pull/1353

**Comment by Youssef1313** [comment]
@kazo0 Do we expect #1353 to get merged soon?

**Comment by Youssef1313** [comment]
Actually, most lines modified here are not modified in #1353

File: src/library/Uno.Material/Styles/Controls/v2/NavigationView.xaml [hunk 1]
```diff
@@ -208,7 +208,7 @@
 
             <StaticResource x:Key="NavigationViewItemRippleFeedback" ResourceKey="PrimaryPressedBrush" />
 
-            <StaticResource x:Key="TopNavigationViewItemForeground" ResourceKey="SystemControlTransparentBrush" />
```
Comment (kazo0) [inline src/library/Uno.Material/Styles/Controls/v2/NavigationView.xaml:211]: We'll need to update the resource docs for all changes resource values:

https://github.com/unoplatform/Uno.Themes/blob/master/doc/styles/NavigationView.md#lightweight-styling

**Comment by Youssef1313** [comment]
@kazo0 Is this ready to merge?

**Review by kazo0 — APPROVED** [review]

**Pull request merged**